relay-sampling/src/lib.rs

```diff
@@ -2296,6 +2443,9 @@ mod tests {
                         start: Some(Utc.ymd(1970, 10, 30).and_hms(0, 0, 0)),
                         end: Some(Utc.ymd(3000, 10, 10).and_hms(0, 0, 0)),
                     },
+                    decaying_fn: DecayingFunction::Linear {
+                        decayed_sample_rate: 0.7,
+                    },
```

Comment:
By changing this helper function, all the tests that use it are now testing `DecayingFunction::Linear` instead of `DecayingFunction::Constant`. Would it not be better to leave this helper function as it was, and test the new linear behavior in another way?

Reply:
The decaying function behavior is tested in the other tests because what we care is to test the sample rate calculation. In this file we test on the other hand the matching logic which doesn't have anything to do with the sample rate.

The `decaying_fn` specified in this test is added because it is required by the struct, I could convert the `Linear` to `Constant` to signal that we don't care about it specifically.

Reply:
👍 thanks!